File: cpp/Keyboard.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class Keyboard
{
private:
    string nama;
    string tipe_keyboard;
    string ukuran;
    string warna;
    string tipe_koneksi;

public:
    Keyboard() {}

    Keyboard(string nama, string tipe_keyboard, string ukuran, string warna, string tipe_koneksi)
    {
        this->nama = nama;
        this->tipe_keyboard = tipe_keyboard;
        this->ukuran = ukuran;
        this->warna = warna;
        this->tipe_koneksi = tipe_koneksi;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    void setTipeKeyboard(string tipe_keyboard)
    {
        this->tipe_keyboard = tipe_keyboard;
    }

    void setUkuran(string ukuran)
    {
        this->ukuran = ukuran;
    }

    void setWarna(string warna)
    {
        this->warna = warna;
    }

    void setTipeKoneksi(string tipe_koneksi)
    {
        this->tipe_koneksi = tipe_koneksi;
    }

    string getNama()
    {
        return nama;
    }

    string getTipeKeyboard()
    {
        return tipe_keyboard;
    }

    string getUkuran()
    {
        return ukuran;
    }

    string getWarna()
    {
        return warna;
    }

    string getTipeKoneksi()
    {
        return tipe_koneksi;
    }

    ~Keyboard() {}
};

File: python/Keyboard.py
class Keyboard:
    def __init__(self, nama="", tipe_keyboard="", ukuran="", warna="", tipe_koneksi=""):
        self.nama = nama
        self.tipe_keyboard = tipe_keyboard
        self.ukuran = ukuran
        self.warna = warna
        self.tipe_koneksi = tipe_koneksi
    
    def setNama(self, nama):
        self.nama = nama
    
    def setTipeKeyboard(self, tipe_keyboard):
        self.tipe_keyboard = tipe_keyboard
    
    def setUkuran(self, ukuran):
        self.ukuran = ukuran
    
    def setWarna(self, warna):
        self.warna = warna
    
    def setTipeKoneksi(self, tipe_koneksi):
        self.tipe_koneksi = tipe_koneksi
    
    def getNama(self):
        return self.nama
    
    def getTipeKeyboard(self):
        return self.tipe_keyboard
    
    def getUkuran(self):
        return self.ukuran
    
    def getWarna(self):
        return self.warna
    
    def getTipeKoneksi(self):
        return self.tipe_koneksi